README.md

```diff
@@ -19,10 +19,12 @@
 
 ## 📦 Packages
 Blade has 2 packages
```

Comment:
remove this line. and we can put the old blade directly without emphasizing over it and putting it as section header?